Assets/MixedRealityToolkit/Utilities/BuildAndDeploy/BuildDeployWindow.cs

```diff
@@ -1,1654 +0,0 @@
-    /// </summary>
-    public class BuildDeployWindow : EditorWindow
-    {
-        #region Internal Types
-
-        private enum BuildDeployTab
-        {
-            UnityBuildOptions,
-            AppxBuildOptions,
-            DeployOptions
-        }
-
-        private enum Architecture
-        {
-            x86 = 0,
-            x64 = 1,
-            ARM = 2,
-        }
-
-        private enum PlatformToolset
-        {
-            Solution,
-            v141,
-            v142,
-        }
-
-        #endregion Internal Types
-
-        #region Constants and Readonly Values
-
-        private const string LOCAL_MACHINE = "Local Machine";
-
-        private const string HOLOLENS_USB = "HoloLens over USB";
-
-        private const string LOCAL_IP_ADDRESS = "127.0.0.1";
-
-        private const string EMPTY_IP_ADDRESS = "0.0.0.0";
-
-        private readonly string[] tabNames = { "Unity Build Options", "Appx Build Options", "Deploy Options" };
-
-        private readonly string[] scriptingBackendNames = { "IL2CPP", ".NET" };
-
-        private readonly int[] scriptingBackendEnum = { (int)ScriptingImplementation.IL2CPP, (int)ScriptingImplementation.WinRTDotNET };
-
-        private readonly string[] deviceNames = { "Any Device", "PC", "Mobile", "HoloLens" };
-
-        private static readonly List<string> Builds = new List<string>(0);
-
-        private static readonly List<string> AppPackageDirectories = new List<string>(0);
-
-        #endregion Constants and Readonly Values
-
-        #region Labels
-
-        private readonly GUIContent buildAllThenInstallLabel = new GUIContent("Build all, then Install", "Builds the Unity Project, the APPX, then installs to the target device.");
-
-        private readonly GUIContent buildAllLabel = new GUIContent("Build all", "Builds the Unity Project and APPX");
-
-        private readonly GUIContent buildDirectoryLabel = new GUIContent("Build Directory", "It's recommended to use 'UWP'");
-
-        private readonly GUIContent useCSharpProjectsLabel = new GUIContent("Generate C# Debug", "Generate C# Project References for debugging.\nOnly available in .NET Scripting runtime.");
-
-        private readonly GUIContent gazeInputCapabilityLabel =
-            new GUIContent("Gaze Input Capability",
-                           "If checked, the 'Gaze Input' capability will be added to the AppX manifest after the Unity build.");
-
-        private readonly GUIContent autoIncrementLabel = new GUIContent("Auto Increment", "Increases Version Build Number");
-
-        private readonly GUIContent versionNumberLabel = new GUIContent("Version Number", "Major.Minor.Build.Revision\nNote: Revision should always be zero because it's reserved by Windows Store.");
-
-        private readonly GUIContent pairHoloLensUsbLabel = new GUIContent("Pair HoloLens", "Pairs the USB connected HoloLens with the Build Window so you can deploy via USB");
```

Comment:
I'm not sure I see this button in the new build window. Is there a new flow for this?
Side-note: I also don't think the old description is accurate. Should be:
>Pairs the USB connected HoloLens with the Build Window so you can deploy _**via WiFi**_

Reply:
Replaced with Test Connection button. The PairHololens didn't actually do anything

Reply:
`Pair HoloLens` grabbed the IP address off a USB-connected HL and added it to the connections list. I don't see that happening with `Test Connection`

Reply:
Ahh so that is what it was trying to do. I was wondering what in the world. So it's really a "Discover Device IP" vs a "Pair Device" because pairing a device with a pin is a totally separate process. 

I'll look at adding a button for this

Reply:
>So it's really a "Discover Device IP" vs a "Pair Device" because pairing a device with a pin is a totally separate process.

Agreed! "Pair Device" definitely overloads with some already existing flows in a confusing way